pkgs/test/cuda/cuda-samples/extension.nix

```diff
@@ -11,6 +11,7 @@ final: prev: let
     "11.4" = "082dkk5y34wyvjgj2p5j1d00rk8xaxb9z0mhvz16bd469r1bw2qk";
     "11.5" = "sha256-AKRZbke0K59lakhTi8dX2cR2aBuWPZkiQxyKaZTvHrI=";
     "11.6" = "sha256-AsLNmAplfuQbXg9zt09tXAuFJ524EtTYsQuUlV1tPkE=";
+    "11.7" = prev.lib.fakeSha256;
```

Comment:
please open an upstream issue for tagging cuda-samples v11.7 and then link to it in a comment here, just so everyone can track why 11.7 gets special treatment here

Reply:
see https://github.com/NVIDIA/cuda-samples/pull/133